### PR TITLE
Added log retrieval in try except

### PR DESCRIPTION
**Pull request by bamachrn**
* this is for for making sure the loop continues even if the log is not available

### CODE REVIEW

File: beanstalk_worker/worker_start_build.py [hunk 1]
```diff
@@ -112,13 +112,19 @@ def start_build(job_details):
         # checking logs for the build phase
         log_command = "oc logs --namespace " + namespace + " build/" + \
             build_details + kubeconfig
-        logs = run_command(log_command)
-        logs = logs[0]
+        try:
+            logs = run_command(log_command)
+            logs = logs[0]
```
Comment (dharmit) [inline beanstalk_worker/worker_start_build.py]: If line #44:

```python
out = p.communicate()
```

is changed to:

```python
out, err = p.communicate()
```

we'll not need to do:
```python
logs = logs[0]
```

**Comment by dharmit** [comment]
LGTM, merging.

**Comment by bamachrn** [comment]
@dharmit rebased and done changes

**Comment by dharmit** [comment]
LGTM, merging.

**Pull request merged**